web_widget_color/__openerp__.py

```diff
@@ -39,6 +39,6 @@
         'static/src/xml/widget.xml',
```

Comment:
Can you include also a README file, following: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst